### ISSUE TITLE
How to nest mixins?

### ISSUE THREAD

**Issue by Dylan-Chapman** [postcss/postcss-mixins#38]
Supposed I have a "between" mixin that takes two breakpoints defined from a map, and creates a media query from them. I know things like media-minmax and custom-media exist, but they don't do precisely what I need.

So, I have code that looks like this:
```css
.test {
    @mixin between md, lg {
        @mixin row;
    }
}
```

This is output as:
```css
@media only screen and (min-width:768px) and (max-width:991px) {
	.test {
		@mixin row
	}
}
```

Is there a way to have the @mixin-content that my "between" mixin spits out (@mixin row, in this scenario) actually be evaluated?

**Comment by ai** [comment]
I try to fix it. But maybe only on this weekend.

**Comment by levibuzolic** [comment]
Just encountered this same issue, my use case is also for a mixin nested inside a mixin that outputs a media query. :+1:

**Comment by Dylan-Chapman** [comment]
Any word on this?

**Comment by ai** [comment]
Sorry, I am total in preparation to dotCSS. But after Friday I will be free.

**Comment by ai** [comment]
Done 7861cbaa65f42cd4b5b30735ec9146be274ef487

**Comment by ai** [comment]
Released in 3.0

**Comment by Dylan-Chapman** [comment]
Works great, thanks!
